README.md

```diff
@@ -83,7 +84,7 @@ var obj = {
 }
 ```
 
--- YOUR ANSWER HERE --
+ES5
```

Comment:
ES6. Google ES6 method shorthand. Not anonymous (it has a name). Call it as `obj.add(33,22);`